### PR TITLE
Add Dockerfile to rhtap.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Fix: https://issues.redhat.com/browse/ACM-7262

### CODE REVIEW

File: Dockerfile.rhtap [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.19 as builder
```
Comment (Kyl-Bempah) [inline Dockerfile.rhtap:1]: You'll want to edit this to point to brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 or whichever go version is in your midstream file.

**Review by Kyl-Bempah — CHANGES_REQUESTED** [review]
You'll want to reference this new file in https://github.com/stolostron/cluster-proxy/blob/main/.tekton/cluster-proxy-plyk-pull-request.yaml#L22 so it knows to use it

File: Dockerfile.rhtap (hunk 1 above)
Comment (xuezhaojun) [reply Dockerfile.rhtap]: Updated.

**Comment by xuezhaojun** [comment]
/assign @zhujian7 
Please take a look! Thanks!

**Pull request merged**